### PR TITLE
Adding output tab to CFN stack viewer

### PR DESCRIPTION
**Pull request by kiiadi**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Description

Screen Shot 2020-06-08 at 1 05 32 PM

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/stack/OutputsView.kt [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+import com.intellij.openapi.Disposable
+import software.amazon.awssdk.services.cloudformation.model.Output
+import software.aws.toolkits.resources.message
+import javax.swing.JComponent
+
+class OutputsTableView : View, OutputsListener, Disposable {
+    private val table = DynamicTableView<Output>(
+        DynamicTableView.Field(message("cloudformation.stack.outputs.key")) { it.outputKey() },
+        DynamicTableView.Field(message("cloudformation.stack.outputs.value")) { it.outputKey() },
```
Comment (awschristou) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/stack/OutputsView.kt:13]: is `it.outputKey()` a copy paste error?

Comment (kiiadi) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/stack/OutputsView.kt:13]: yup!

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/stack/EventsTable.kt [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+private class StatusCellRenderer : DefaultTableCellRenderer() {
+    override fun getTableCellRendererComponent(table: JTable, value: Any, isSelected: Boolean, hasFocus: Boolean, row: Int, column: Int): Component =
+        super.getTableCellRendererComponent(table, value, isSelected, hasFocus, row, column).also {
+            it.foreground = StatusType.fromStatusValue(value as String).color
+        }
+}
+
+internal class EventsTableImpl : EventsTable, Disposable {
```
Comment (abrooksv) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/stack/EventsTable.kt:38]: we don't dispose anything, am i missing us using this as a parent disposable?

**Comment by abrooksv** [comment]
Still need to do a closer look, what does Copy support look like? Does it copy the whole row? Nothing?

Do we want more specific copy actions? Like Copy Export Name? Copy Resource Arn?

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/stack/EventsTable.kt (hunk 2 above)
Comment (kiiadi) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/stack/EventsTable.kt:38]: it's the pattern that exists in the other table (`software.aws.toolkits.jetbrains.services.cloudformation.stack.EventsTableImpl`

**Comment by kiiadi** [comment]
> Still need to do a closer look, what does Copy support look like? Does it copy the whole row? Nothing?
> 
> Do we want more specific copy actions? Like Copy Export Name? Copy Resource Arn?

? copy support? just copies the row...

**Comment by abrooksv** [comment]
> > Still need to do a closer look, what does Copy support look like? Does it copy the whole row? Nothing?
> > Do we want more specific copy actions? Like Copy Export Name? Copy Resource Arn?
> 
> ? copy support? just copies the row...

Couldn't remember if we got that for free or not.

File: jetbrains-core/tst/software/aws/toolkits/jetbrains/services/cloudformation/stack/DynamicTableViewTest.kt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+package software.aws.toolkits.jetbrains.services.cloudformation.stack
+
+import org.junit.Assert.*
+
+class DynamicTableViewTest
```
Comment (abrooksv) [inline jetbrains-core/tst/software/aws/toolkits/jetbrains/services/cloudformation/stack/DynamicTableViewTest.kt:5]: Why the empty test class?

Comment (kiiadi) [reply jetbrains-core/tst/software/aws/toolkits/jetbrains/services/cloudformation/stack/DynamicTableViewTest.kt:5]: accidental commit that's why.

**Pull request merged**